frontend/src/lib/components/HedgehogBuddy/HedgehogOptions.tsx

```diff
@@ -1,79 +1,165 @@
-                    onChange={setInteractWithElements}
-                    tooltip="If enabled the Hedgehog might land on elements of the application, otherwise they will always land on the ground"
-                />
-                <LemonSwitch
-                    bordered
-                    label="Keyboard controls (WASD / arrow keys)"
-                    checked={keyboardControlsEnabled}
-                    onChange={setKeyboardControlsEnabled}
-                    tooltip="If enabled you can use the WASD or arrow key + space to move around and jump."
-                />
+            <div className="space-y-2">
+                <h4>Options</h4>
+                <div className="flex items-center gap-2 flex-wrap">
+                    <LemonSwitch
+                        bordered
+                        label="Walk around freely"
+                        checked={hedgehogConfig.walking_enabled}
+                        onChange={(val) =>
+                            patchHedgehogConfig({
+                                walking_enabled: val,
+                            })
+                        }
+                        tooltip="If enabled the Hedgehog will walk around the screen, otherwise they will stay in one place. You can still move them around by dragging them."
+                    />
+                    <LemonSwitch
+                        bordered
+                        label="Interact with elements"
+                        checked={hedgehogConfig.interactions_enabled}
+                        onChange={(val) =>
+                            patchHedgehogConfig({
+                                interactions_enabled: val,
+                            })
+                        }
+                        tooltip="If enabled the Hedgehog might land on elements of the application, otherwise they will always land on the ground"
+                    />
+                    <LemonSwitch
+                        bordered
+                        label="Keyboard controls (WASD / arrow keys)"
+                        checked={hedgehogConfig.controls_enabled}
+                        onChange={(val) =>
+                            patchHedgehogConfig({
+                                controls_enabled: val,
+                            })
+                        }
+                        tooltip="If enabled you can use the WASD or arrow key + space to move around and jump."
+                    />
+                    <LemonSwitch
+                        bordered
+                        label="Party mode"
+                        checked={hedgehogConfig.party_mode_enabled}
+                        onChange={(val) =>
+                            patchHedgehogConfig({
+                                party_mode_enabled: val,
+                            })
+                        }
+                        tooltip="If enabled then all of your organization members will appear as hedgehogs as well!"
```

Comment:
This feels like it could do with a better description. I wasn't sure if this just toggled things for me or everyone in my org